Makefile

```diff
@@ -78,7 +78,7 @@ start: compile-native compile-port
 
 # Run an interactive terminal with the main supervisor setup.
 iex: compile-native compile-port
-	iex -S mix run -- --checkpoint-sync https://beaconstate-mainnet.chainsafe.io
+	iex -S mix run -- --checkpoint-sync https://sync-mainnet.beaconcha.in/
```

Comment:
changed this because the previous one was returning a state that finalized 3hs ago, vs the new one that returns one finalized 20 min ago.